app/client/src/components/projects/ProjectKPIsForm.tsx

```diff
@@ -15,28 +15,14 @@
 //
 
 import React from 'react';
-import { useFormik } from 'formik';
-import {
-  Box,
-  Container,
-  FormControl,
-  InputLabel,
-  MenuItem,
-  Select,
-  Typography,
-  TextField,
-  Button,
-} from '@material-ui/core';
+import { Container, FormControl, Typography, Button } from '@material-ui/core';
 // import DateFnsUtils from '@date-io/date-fns';
-import { useHistory } from 'react-router-dom';
-import useApi from '../../utils/api';
-import { validateNewProject } from '../../utils/validationSchema';
 
 const ProjectKPIsForm: React.FC = () => {
-
   const handleNewMilestone = () => {
```

Comment:
TODO: Change this method name to handleNewKPI; SF forgot to update this method when copying code over initially.

Reply:
We can fix it when implementing the KPI form.